pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -84,26 +82,17 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 			continue
 		}
 
-		namespace := api.NamespaceName(job.Namespace)
-		queueMap, found := jobsMap[namespace]
-		if !found {
-			namespaces.Push(namespace)
-
-			queueMap = make(map[api.QueueID]*util.PriorityQueue)
-			jobsMap[namespace] = queueMap
-		}
+		queues.Push(ssn.Queues[job.Queue])
```

Comment:
It seems that it will push duplicate queues to `queues`

Reply:
Yes, queue should be pushed in `queues` only on the first job in the queue is picked. I'll modify it, thank you.